src/send_kcidb.py

```diff
@@ -46,6 +50,34 @@ def __call__(self, configs, args):
 
         sys.stdout.flush()
 
+    def get_revision(self, node):
+        return {
+            "builds": [],
+            "checkouts": [
+                {
+                    "id": node["_id"],
+                    "origin": "kernelci",
+                    "tree_name": node["revision"]["tree"],
+                    "git_repository_url": node["revision"]["url"],
+                    "git_commit_hash": node["revision"]["commit"],
+                    "git_repository_branch": node["revision"]["branch"],
+                    "start_time": node["created"],
+                },
+            ],
+            "tests": [],
+            "version": {
+                "major": 4,
+                "minor": 0
+            }
+        }
+
+    def send_revision(self, node):
+        project_id = os.getenv('PROJECT_ID')
+        topic_name = os.getenv('TOPIC_NAME')
+        client = Client(project_id=project_id,
+                        topic_name=topic_name)
```

Comment:
We need to avoid creating the client each time we need to submit something. It takes a long time for the client to authenticate and connect to the message queue. The best approach would be to create it only once, when the program starts.

Reply:
Okay 
I will update it